### DISCUSSION TITLE
How soon until a new release of Docker image to address php vulnerability CVE-2024-4577 (simple rebuild/push will fix it)

### DISCUSSION THREAD

**Discussion by frachel** [ltb-project/self-service-password#911] [Q&A]
The vulnerability is for php versions < 8.2.20. Current release of ssp is on 8.2.18.

The parent image used in the Dockerfil, php-8.2-apache, has been updated so really just need to rebuild the docker image and push a new version.

Any chance that can be expedited? We had to stop our use of the tool as per our security team, and would prefer not to build/extend the official image to update the php version.

**Comment by coudot** [comment]
Done this morning
